mongo_connector/oplog_manager.py

```diff
@@ -329,7 +329,15 @@ def run(self):
                                 elif operation == 'c':
                                     # use unmapped namespace
                                     doc = entry.get('o')
-                                    docman.handle_command(doc,
+                                    if doc.get('renameCollection'):
+                                        db = entry['ns'].split('.', 1)[0]
+                                        db, coll = docman.command_helper.map_collection(db, doc['renameCollection'])
+                                        if doc and coll:
+                                            docman.handle_command(doc,
```

Comment:
Could probably be made slightly nicer by removing the else line and adding a continue *if not doc or not coll*